tests/config/browserTest.ts

```diff
@@ -57,6 +58,18 @@ type BrowserTestTestFixtures = PageTestFixtures & {
 };
 
 const test = baseTest.extend<BrowserTestTestFixtures, BrowserTestWorkerFixtures>({
+  loopback: [async ({ channel }, use) => {
```

Comment:
If I were to run `webServer`, how should I test `localhost:3000` in my playwright tests as a user?